recipes/xover/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "xover" %}
+{% set version = "2.0.0" %}
+{% set python_min = "3.10" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default.

```suggestion
```